src/EditorFeatures/Core/Implementation/TodoComment/TodoCommentOptions.cs

```diff
@@ -6,9 +6,7 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.TodoComments
 {
     internal static class TodoCommentOptions
     {
-        public const string OptionName = "TaskList/Tokens";
-
         [ExportOption]
-        public static readonly Option<string> TokenList = new Option<string>(OptionName, "Token List", defaultValue: "HACK:1|TODO:1|UNDONE:1|UnresolvedMergeConflict:0");
+        public static readonly Option<string> TokenList = new Option<string>(nameof(TodoCommentOptions), nameof(TokenList), defaultValue: "HACK:1|TODO:1|UNDONE:1|UnresolvedMergeConflict:0");
```

Comment:
No, the list is just empty. It's a list, since some options might be serialized at project vs. document vs. user levels.